pkg/server/server.go

```diff
@@ -91,12 +93,23 @@ func New(configuration options.Configuration) (controllerruntime.Manager, error)
 		return nil, err
 	}
 
+	// If configuration has a TargetLabelSelector, then read the values
+	// and create the labels.Selector object.
+	var targetLabelSelector labels.Selector
+	logging.Logger.Info("Configuration targetLabelSelector text is: " + configuration.TargetLabelSelector)
+	if configuration.TargetLabelSelector != "" {
```

Comment:
Actually I would see behavior here as
1. Initialize `labelSelector` as `labels.Everything()`
2. If there is `targetLabelSelector` set, override with the setting
I guess it would simplify the flow

```
targetLabelSelector := labels.Everything()
if (configuration.targetLabelSelector !="") {
  // read from config
}
```

Reply:
Thanks @grzesuav . I have updated accordingly based on your suggestion.